pillar.example

```diff
@@ -85,6 +88,29 @@ vhosting:
         #    http2: True
         #    letsencrypt: True
 
+        # Pagespeed example
+        #pagespeed-example.com:
+        #  pagespeed:
+        #    enable: True
+
+        # Pagespeed example with different cache path
+        #pagespeed-example-path.com:
+        #  pagespeed:
+        #    enable: True
+        #    file_cache_path: '/tmp/pagespeed'
+
+        # Pagespeed example with specific config
+        #pagespeed-example-config.com:
+        #  pagespeed:
+        #    enable: True
+        #    config:
+        #      DisableFilters: 'rewrite_images,combine_css'
+
+        # Add extra config to the location section (nginx only)
+        extra-location-config.com:
```

Comment:
Should this config thing also be in comments since it's an example?

Reply:
Any vhost configs that are testable are enabled in this file. Only the one's that aren't possible such as LetsEncrypt (due to domain ownership check) and Pagespeed (due to missing module not available in the public repositories) are commented out to serve as an example. But technically the whole file _is_ an example :-)

Reply:
👍